### ISSUE TITLE
Fix headings in Explore Data

### ISSUE THREAD

**Issue by cthomasONRR** [DOI-ONRR/nrrd#2858]
I'm trying the separate the Explore Data issues in #2732 into their own issue and better explain. Both Maroya and I were getting a little confused on everything that needs to be done.

- [ ] Card labels are not a heading but should be H3 (sections on cards are H4) 
@eelzi-ONRR Does the screengrab below explain the issue location?
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaGpyIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--dd1cb7e404f3b61d87c89e7d30c873951dd15ef1/image.png)

- [ ] second level headings on map cards are H6, but the second level headings on Compare cards are H4
 [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaHZyIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--58a930d16f1519d813a37d8722df8ae38c16ecde/image.png)

**Comment by cthomasONRR** [comment]
I created issue #2859 for headings issues that probably require some design solutions not just updating code.

**Comment by lpgoldstein** [comment]
No longer relevant
